src/common/syncjournaldb.cpp

```diff
@@ -2031,6 +2031,22 @@ QByteArrayList SyncJournalDb::conflictRecordPaths()
     return paths;
 }
 
+QByteArray SyncJournalDb::conflictFileBaseName(const QByteArray &conflictName)
+{
+    auto conflict = conflictRecord(conflictName);
+    QByteArray result;
+    if (conflict.isValid()) {
+        getFileRecordsByFileId(conflict.baseFileId, [&result](const SyncJournalFileRecord &record) {
+            if (!record._path.isEmpty())
+                result = record._path;
+        });
+    }
+
+    if (result.isEmpty())
```

Comment:
I would use {} even if it is just one line.

Reply:
Done